tests/testthat/_snaps/asis-nodes.md

```diff
@@ -3,7 +3,7 @@
     Inline math delimiters are not balanced.
     
     HINT: If you are writing BASIC code, make sure you wrap variable
-          names and code in backtics like so: `INKEY$`. 
+          names and code in backtics like so: `INKEY$`.
```

Comment:
the hint in the code also doesn't have this whitespace :thinking:

Reply:
No worries! I think your editor trimmed the trailing whitespace of [asis-nodes.R#L13](https://github.com/ropensci/tinkr/pull/88/files#diff-60bef5af43a1bd541299ce9bceeb226afb23deaa5e637b145c85e05ffd552b0cL13-R13) when you updated the stringr function.